tests/map/map.test.ts

```diff
@@ -159,20 +167,21 @@ describe('map markers', () => {
         // render the markers.
         MAP['_options'].x.scale.value = 'log';
 
-        // use setTimeout with a timeout of 0 to allow the browser to re-render
-        // the marker before checking its position
-        setTimeout(() => {
-            position = getMarker(firstGUID).marker.getBoundingClientRect();
-            assert(position.x !== initialPosition.x);
-            assert(position.y === initialPosition.y);
-        }, 0);
+        // Use setTimeout with a timeout of 0 to let Plotly re-render and
+        // then trigger an update of the marker before checking its position
+        await wait();
```

Comment:
Yes I agree, what I meant is that we don't need `setTimeout()` for `getBoundingClientRect()` (as calling it always triggers a repaint, so the change would be visible immediately) but because the marker update is only triggered after Plotly is done rendering, and Plotly probably relies on `requestAnimationFrame()`. So the delay comes from Plotly, not from the browser. Does this make sense? I can update the comment to make it more clear.

Reply:
Yes, this makes more sense! If you can update the comment to say something like this (and maybe update the function name as well), that would be great!